### PR TITLE
Change nav-bar fixed to static to fix overlay

### PR DESCRIPTION
**Pull request by thePaulista**
1. Changed nav-bar from 'fixed' to 'static' to adjust the navbar overlay
2. Flash message moved to partial from application.layout to a _flashes.html.erb file

### CODE REVIEW

**Comment by JaredRoth** [comment]
@thePaulista: Could you summarize what the difference is between these lines?

New flash (in partial):
<%= content_tag(:p, sanitize(value), :class => "flash-#{key}") %>

Old flash (in application.html.erb):
<%= content_tag :div, message.html_safe, class: type %>

**Comment by thePaulista** [comment]
either one works.  they do the same exact thing.  .html_safe and sanitize
have similar functionality (sal and i experimented with those doing the
little shop).  value, b/c it's coming in as a hash for :notice => and the
message. i think that the "flash-#{key} was something i had taken from the
carmer lecture.  the rendering is pretty much the same.  also the div tag
vs p didn't make any difference.  it was just a piece of code i put on my
little shop project. either one ends up being the same.

**Comment by JaredRoth** [comment]
Ok, thank you for that explanation. I think it may be more clear if we change key/value back to type/message for the sake of being explicit.

That can be done later, though, so I am approving the pull.
